.github/CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing Guidelines
 
--   You can join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). Each active repo has its own stream to direct questions to (for example #powerup or #portal). Open Source Programs stream is [#open-source-progs](https://anitab-org.zulipchat.com/#streams/237907/open-source-progs).
+-   You can join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). Each active repo has its own stream to direct questions to (for example #powerup or #portal). Open Source Programs stream is [#open-source-progs](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs).
```

Comment:
You can remove the `(for example #powerup or #portal)` since they seems to be missing on channels. Instead replace it with `Each active repo has its own stream to direct questions (labeled as #question) on Zulip.` This will make new contributors to easily understand find the channels to ask their doubts. 😄